### DISCUSSION TITLE
How do you draw a transparent image?

### DISCUSSION THREAD

**Discussion by nomad-software** [hajimehoshi/ebiten#2470] [Q&A] [answered]
How do you draw a transparent image?
For example, I have a coloured background and I want to draw a sprint on top at 50% opacity.

**Comment by hajimehoshi** [comment, marked as answer]
Try `ColorM`'s `Scale`.

https://github.com/hajimehoshi/ebiten/blob/2.4/examples/alphablending/main.go

**Comment by nomad-software** [comment]
For reference, the part that was really confusing me was solved by using this as the background colour:

```
backgroundColour = color.NRGBA{0, 0, 0, 255}
```

I guess the background colour is important for the image to alpha-blend properly.
